Move constraints from the Admin section to top of nav

davidoliverSP2 opened the pull request:
Depends on https://github.com/neo-technology/neo4j-manual-modeling/pull/2152

davidoliverSP2 committed:
move contraints folder

davidoliverSP2 committed:
move files and folders

davidoliverSP2 committed:
rename files

davidoliverSP2 committed:
change includes to have right path

davidoliverSP2 committed:
add abstracts and get the level offset correct

davidoliverSP2 committed:
resolve build warnings

davidoliverSP2 committed:
restructure the syntax page

davidoliverSP2 committed:
fix broken links and update header levels

Hunterness commented on cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc:
```diff
@@ -0,0 +1,143 @@
+
+[source, cypher, role=noplay]
+----
+SHOW [ALL\|UNIQUE\|NODE [PROPERTY] EXIST[ENCE]\|RELATIONSHIP [PROPERTY] EXIST[ENCE]\|[PROPERTY] EXIST[ENCE]\|NODE KEY] CONSTRAINT[S]
```
While I remember it, I found a tiny error in this syntax line:
```suggestion
SHOW [ALL\|UNIQUE\|NODE [PROPERTY] EXIST[ENCE]\|REL[ATIONSHIP] [PROPERTY] EXIST[ENCE]\|[PROPERTY] EXIST[ENCE]\|NODE KEY] CONSTRAINT[S]
```

Hunterness commented on cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc:
```diff
@@ -0,0 +1,143 @@
+[[administration-constraints-syntax]]
```
should all of these links be updated to not include `administration`? (also goes for the links in the index and examples files)

davidoliverSP2 replied:
Yes, I think so, but I think this should be handled in another PR, post publication

Hunterness commented on cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc:
```diff
@@ -0,0 +1,143 @@
+
+[source, cypher, role=noplay]
+----
+SHOW [ALL\|UNIQUE\|NODE [PROPERTY] EXIST[ENCE]\|RELATIONSHIP [PROPERTY] EXIST[ENCE]\|[PROPERTY] EXIST[ENCE]\|NODE KEY] CONSTRAINT[S]
+    [YIELD { * \| field[, ...] } [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+    [WHERE expression]
+    [RETURN field[, ...] [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
```
If it is no longer in a table we don't need to escape the pipes, `\|`

Hunterness commented on cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala:
```diff
@@ -113,7 +113,7 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
         """
           |One way of filtering the output from `SHOW CONSTRAINTS` by constraint type is the use of type keywords,
-          |listed in the <<administration-constraints-syntax, syntax table>>.
+          |listed in the <<administration-constraints-syntax-list, _Syntax for listing constraints_ example>>.
```
not sure `example` is the correct word here

Hunterness committed:
Fill out the syntax section a bit more since it is no longer a table

Also moved in the list of returned columns for SHOW CONSTRAINTS
from examples to syntax section, if feels like it belongs more there.

davidoliverSP2 committed:
add check marks to table

davidoliverSP2 committed:
reword to remove the word example

davidoliverSP2 committed:
update admin paragraph
